lib/response.js

```diff
@@ -14,6 +14,8 @@ exports = module.exports = internals.Response = function (req, onEnd) {
 
     Http.ServerResponse.call(this, { method: req.method, httpVersionMajor: 1, httpVersionMinor: 1 });
 
+    this.shot = { trailers: {}, payloadChunks: [] };
```

Comment:
The `response._headers` is keeping the contract with the Node object, and I didn't see the need to duplicate them.